Fix125

becega opened the pull request:
fix #125 
fix #108 
fix #76 
fix #74

Commit:
Updated Create Test to configure a static module
Updated Pannels on the test to include pass value.

Commit:
updated the settings (disable the test during IC as they will pas when after scripted)

DCAF-Builder commented:
Bleep bloop!

LabVIEW Diff Robot here with some diffs served up hot for your pull request.

Notice something funny? Help fix me on [my GitHub repo.](https://github.com/LabVIEW-DCAF/buildsystem)


### User Control Module Test.lvlib--Create Test Configuration.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/10%3A29%3A29/User%20Control%20Module%20Test.lvlib--Create%20Test%20Configuration.vi.png)

### User Control Module Test.lvlib--Runtime Initialization Test.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/10%3A29%3A29/User%20Control%20Module%20Test.lvlib--Runtime%20Initialization%20Test.vi.png)

### User Control Module Test.lvlib--runtime to editor crosslink test.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/10%3A29%3A29/User%20Control%20Module%20Test.lvlib--runtime%20to%20editor%20crosslink%20test.vi.png)

### User Control Module Test.lvlib--Serialization Test.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/10%3A29%3A29/User%20Control%20Module%20Test.lvlib--Serialization%20Test.vi.png)

jacobson3 commented:
The "Runtime Initialization Test.vi" has a pass boolean added but it it was never connected to anything. I'll go ahead and make that change because I want to add comments and a VI description to all of the VIs attached.

jacobson3 committed:
Minor Static Module Test Fixes

- Made sure all VIs changed had comments and VI descriptions
- Crosslink lvtest now checks pass boolean
- Runtime Initialization test has pass properly wired now

DCAF-Builder commented:
Bleep bloop!

LabVIEW Diff Robot here with some diffs served up hot for your pull request.

Notice something funny? Help fix me on [my GitHub repo.](https://github.com/LabVIEW-DCAF/buildsystem)


### tag bus configurable plugin editor node.lvclass--Edit Line.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/15%3A52%3A28/tag%20bus%20configurable%20plugin%20editor%20node.lvclass--Edit%20Line.vi.png)

### User Control Module Test.lvlib--config crosslink test.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/15%3A52%3A28/User%20Control%20Module%20Test.lvlib--config%20crosslink%20test.vi.png)

### User Control Module Test.lvlib--Create Test Configuration.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/15%3A52%3A28/User%20Control%20Module%20Test.lvlib--Create%20Test%20Configuration.vi.png)

### User Control Module Test.lvlib--Runtime Initialization Test.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/15%3A52%3A28/User%20Control%20Module%20Test.lvlib--Runtime%20Initialization%20Test.vi.png)

### User Control Module Test.lvlib--runtime to editor crosslink test.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/15%3A52%3A28/User%20Control%20Module%20Test.lvlib--runtime%20to%20editor%20crosslink%20test.vi.png)

### User Control Module Test.lvlib--Serialization Test.vi.png: ![capture](https://raw.githubusercontent.com/DCAF-Builder/diff-pics/master/LabVIEW-DCAF/DCAF-Project-Templates/PR-126/2018-08-17/15%3A52%3A28/User%20Control%20Module%20Test.lvlib--Serialization%20Test.vi.png)

jacobson3 reviewed (approved):
Looks good, had to add pass boolean to Crosslink lvtest and have Runtime Initialization Test properly pass out pass boolean.
